cirq/google/api/v2/program.py

```diff
@@ -93,7 +93,7 @@ def grid_qubit_from_proto_id(proto_id: str) -> 'cirq.GridQubit':
     Raises:
         ValueError: If the string not of the correct format.
     """
-    parts = proto_id.split('_')
+    parts = proto_id.replace("q","").split('_')
```

Comment:
We really only need to strip a `q` prefix, so I wonder if we should be more specific here. We don't want to accept strange forms like `1_2q3`, for example.
```python
if proto_id.startswith('q'):
    proto_id = proto_id[1:]
parts = proto_id.split('_')
```
Another possibility would be to rewrite the function use a regex like `r'q?(-?\d+)_(-?\d+)'`.

Reply:
Sounds good - the narrower the better!